.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+      - name: golangci-lint
```

Comment:
Can this be scoped to changes only to *.go files?

Reply:
> > Would it be possible to do a write up for IDE support to go alongside adding linters?
> 
> Not sure where would that go. Also the configuration is different for each IDE. We can plan to add a dev section on the repos describing about the environment. Like for example go version used in the projects, the linters and other things. I believe we can have a follow-up issue for that as this issue is for adding linter so I am not sure if it makes sense to write the whole dev-doc in this. But definitely I can write a dev-doc on a separate PR, wdyt?

Would go wherever else we have setup guidelines. I think it could be a really valuable investment. Agree it would be different for each IDE but you could cover yours, for example. @moshloop @BrendanGalloway Do you think adding an IDE doc would be best done here or in follow up?